react/components/Metrics/Controllers/TimeControllers/Relative/index.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+
+const Relative = () => {
+  return (
+    <div className="pa4 mh2" >
+      Comming soon!
```

Comment:
`Comming` => `Coming`. Also, this is not internationalized.

Reply:
Ok